index.src.html

```diff
@@ -22,6 +22,9 @@
     text: location; url: concept-css-style-sheet-location
     text: owner node; for: CSSStyleDeclaration; url: cssstyledeclaration-owner-node
     text: owner node; for: CSSStyleSheet; url: concept-css-style-sheet-owner-node
+spec: Fetch; urlPrefix: https://fetch.spec.whatwg.org/
+  type: dfn
+    text: ABNF; url: abnf
```

Comment:
I don't think this is a good idea. Fetch doesn't define ABNF, RFC5234 does. Instead, I'd suggest that you add something like the text from https://github.com/w3c/webappsec-csp/commit/40d2e607855b91f91cfb0f1a183be1b3ce209553 which explicitly points to RFC7230's `#rule` definition, which is all that you're using here.

@annevk: Perhaps this is something we could move to INFRA?